pkg/settings/dvm.go

```diff
@@ -54,3 +61,14 @@ func (r *RsyncOpts) Load() error {
 	}
 	return err
 }
+
+// Load loads DVM options
+func (r *DvmOpts) Load() error {
+	var err error
+	r.EnablePVResizing = getEnvBool(EnablePVResizing, false)
```

Comment:
This feels like it should be a Plan option to me, just because PV discovery happens on the plan. Feel free to tell me if you think otherwise.

Reply:
@djwhatle There are two kinds of flags here:
1. Enable/Disable discovering PV usage statistics **(Enabled by default)**
    -  When Set, it allows letting users know that the PVs will result in errors if they proceed to migrate without instructing DVM to use the corrected sizes
    - When Unset, it allows us to disable the feature, this is used as an escape hatch if we see issues with this feature in customer environment. We will remove this flag in future releases. 
    - this is not advertised until users run into any issues
2. Enable/Disable PV resizing in DVM controller **(Disabled by default)**
    - When Set, DVM controller will use the proposed sizes
    - When Unset, DVM controller will take no action on proposed sizes
    - This is the flag we will let users know about for 1.4.1, in case they have volumes resulting in warnings

Reply:
Thanks for the overview. I was thinking that if we were to enable this for indirect migration down the road, it would make more sense for the resize flag to exist on plan. If we get to that point we can always move it. Seems like we may never implement for indirect migration.

Reply:
@djwhatle Thats the plan. When we have UI change for toggling the option from MigPlan, we are planning to move this flag to plan level.

Reply:
+1, marking as resolved.